packages/neuron-wallet/src/controllers/wallets.ts

```diff
@@ -95,6 +96,29 @@ class WalletsController {
     }
   }
 
+  public static create({
+    name,
+    password,
+    mnemonic,
+    receivingAddressNumber = 17,
```

Comment:
I have thought we've changed `17` and `3` already?

Reply:
updated in https://github.com/nervosnetwork/neuron/pull/304/commits/99e22208326928df20032a786382f390d033c283